models/enterprise_sonic/l2_acls/merged_example_01.txt

```diff
@@ -0,0 +1,86 @@
+# Using merged
+#
+# Before State:
+# -------------
+#
+# sonic# show running-configuration mac access-list
+# !
+# mac access-list test
+#  seq 1 permit host 22:22:22:22:22:22 any vlan 20
+# sonic#
+
+  - name: Merge provided Layer 2 ACL configurations
+    dellemc.enterprise_sonic.sonic_l2_acls:
+      config:
+        - name: 'test'
+          rules:
+            - sequence_num: 2
+              action: 'permit'
+              source:
+                any: true
+              destination:
+                any: true
+              ethertype:
+                value: '0x88cc'
+              remark: 'LLDP'
+            - sequence_num: 3
+              action: 'permit'
+              source:
+                any: true
+              destination:
+                address: '00:00:10:00:00:00'
+                address_mask: '00:00:ff:ff:00:00'
+              pcp: 'ca'
+              pcp_mask: 3
+            - sequence_num: 4
+              action: 'deny'
+              source:
+                any: true
+              destination:
+                any: true
+              vlan_tag_format:
+                multi_tagged: true
+        - name: 'test1'
+          remark: 'test_mac_acl'
+          rules:
+            - sequence_num: 1
+              action: 'permit'
+              source:
+                host: '11:11:11:11:11:11'
+              destination:
+                any: true
+            - sequence_num: 2
+              action: 'permit'
+              source:
+                any: true
+              destination:
+                any: true
+              ethertype:
+                arp: true
+              vlan_id: 100
+            - sequence_num: 3
+              action: 'deny'
+              source:
+                any: true
+              destination:
+                any: true
+              dei: 0
+      state: merged
+
+# After State:
+# ------------
+#
+# sonic# show running-configuration mac access-list
+# !
+# mac access-list test
+#  seq 1 permit host 22:22:22:22:22:22 any vlan 20
+#  seq 2 permit any any 0x88cc remark LLDP
+#  seq 3 permit any 00:00:10:00:00:00 00:00:ff:ff:00:00 pcp ca pcp-mask 3
```

Comment:
Is there some version of SONiC that supports pcp-mask as a CLI configuration attribute?

Reply:
Recent SONiC versions support pcp-mask for L2 ACL rule.

Reply:
The `pcp` spec has been updated with sub-options `value`, `mask` and `traffic_type` to allow specifying the PCP values both as an integer and a string.
The examples have also been updated based on the new `pcp` spec.

Reply:
Verified. These updates look good. 

The associated issue is "resolved".